governance.md

```diff
@@ -1,149 +1,285 @@
+  - Expectations
+    - review PRs
+    - fix bugs
+    - maintain and improve health and quality of the project
+    - provide user support
+    - fulfill other project responsibilities and roles
+    - mentor APPROVERS, REVIEWERS, and CONTRIBUTORS
+  - Benefits
+    - added to the repo’s maintainers github team (e.g., [`kubernetes-maintainers`](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers))
+    - write access to the repo: assign issues/PRs, add/remove labels and milestones, 
+      edit issues and PRs, edit wiki, create/delete labels and milestones
+    - technically can approve any PR and cause it to be merged by the submit queue, but expected to
+      respect OWNERS files
+- **OWNER**:
+  - includes all requirements, expectations, and benefits of MAINTAINERS
+  - Requirements
+    - PR and tenure requirements and nomination process TBD
+    - familiar enough with some area/component to be in its top-level
+      [OWNERS](contributors/devel/owners.md) file `owners` list
+    - may be asked to become an area/component OWNER as a precondition for accepting the 
+      contribution of a new component or other major function
+    - TBD: which areas/components need OWNERS? Examples: Apiserver, Kubelet, Kubectl
+  - Expectations
+    - mentor APPROVERS, REVIEWERS, and CONTRIBUTORS
+  - Benefits
+    - design/proposal approval authority for some area of the project, though escalation to LEADS
+      is still possible, and beta/GA APIs must still be vetted by API REVIEWERS and APPROVERS
+- **LEAD**:
+  - includes all requirements, expectations, and benefits of OWNERS
+  - Requirements
+    - current de facto project leads have been involved with the project since at least June 2014, and
+      are a subset of people from the `approvers` list in the top-level OWNERS file in the `kubernetes` repo
+    - demonstrated strong technical vision and judgement
+    - demonstrated empathy for the user and open-source developer perspective
+    - committed to project's mission and culture
+    - spends majority of time on Kubernetes
+  - Expectations
+    - provide overall technical guidance and vision for the project
+    - maintain the [definition of the project](https://kubernetes.io/docs/whatisk8s/)
+    - decide project structure, such as system layers (e.g., core) and repository breakdown
+    - assist SIGs in identifying areas of overlapping technical and ownership responsibility
+    - resolve technical escalations in the cases of OWNER and SIG lead disagreements 
+  - Benefits
+    - project decision makers
+    - technically can approve virtually any PRs
+    - can [Sponsor incubator repos](incubator.md)
+    - can Sponsor MAINTAINERS
+
+## Orthogonal technical roles
+
+API REVIEWER and APPROVER are called out specifically because the API is critical to the
+identity of Kubernetes and is a horizontal area that crosses directories and SIGs.
+
+- [**API REVIEWER**]:
```

Comment:
@bgrant0607 -- I think that breaking this apart now will result in more people reading it and getting better feedback.  It is a wall of text and, as such, limits the number of people that are paying attention.  We need to make sure we communicate well in addition to getting the details right.

@thockin -- I don't find this unambiguous at all.  Right now there is the k8s API but, as we have more projects in the kubernetes ecosystem they will have their own APIs.  We need to define the scope for pretty much everything in this document as to if it just applies to kubernetes/kubernetes or to other things that are in kubernetes/* or some other way to describe this.

Are we going to have completely different ranks for `kubernetes/api-machinery`? What about`kubernetes/kops`?